fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/ContainerServiceImpl.java

```diff
@@ -61,10 +63,22 @@ public Container findOrCreate(final Session session, final String path) {
 
             if (node.isNew()) {
                 initializeNewObjectProperties(node);
+
+                final Node parent = node.getDepth() > 0 ? node.getParent() : null;
+
+                if (parent != null) {
+                    touch(parent);
+                    touchLdpMembershipResource(node);
+                }
+            }
+
+            if (node.isNew()) {
+                touch(node);
             }
 
             return new ContainerImpl(node);
         } catch (final RepositoryException e) {
+            LOGGER.info("Error: {}", e);
```

Comment:
@acoburn In this particular case, I think it's fine to just rethrow the exception and it'll be handled elsewhere.  But I'm responding to the general issue that's come up several times in code review: developers would like to be able to see stacktraces, but we frequently ask them to remove them in code review for the reasons I mentioned.

What I'm thinking of specifically is being able to add a System property (`-Dfcrepo.debug=true` or something like that) to output stacktraces any time we normally log exceptions.  This would probably involve some kind of logging helper with Spring config to enable/disable the stacktrace logging.